spec/apply_for_a_course_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe "Apply for a course", type: :feature do
+  scenario "User can apply for a course" do
+    visit "https://qa.find-postgraduate-teacher-training.service.gov.uk/course/2Z3/B275"
+    click_on "Apply for this course"
+
+    expect(page).to have_content("Do you already have an account?")
+    choose "Yes, sign in"
+    fill_in "Email address", with: "ollie.treend@education.gov.uk"
```

Comment:
Might be worth moving this to the `.env` file. So you don't expose your email address to the world.

Reply:
[Moved to `.env.example`](https://github.com/DFE-Digital/bat-e2e-tests/compare/950305089d461d805c5924da36f13c662852ed35..a8bc1c147f452df60869603d4a7cdc54c3c38114)